### PR TITLE
Upgrade versions

### PR DESCRIPTION
**Pull request by andreysaksonov**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-    - $HOME/.m2
+os:
+  - linux
+sudo: required
 language: java
 jdk:
   - openjdk11
 services:
   - docker
+before_install:
```
Comment (andreysaksonov) [inline .travis.yml]: I took this from https://github.com/testcontainers/testcontainers-scala/blob/master/.travis.yml#L23, looks like shipped with Ubuntu 16.04 (used by jdk11 image on travis) docker-compose is outdated

File: tcp-mocker-app-test/tcp-mocker-app/docker-compose.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: "2"
+version: '2.1'
```
Comment (andreysaksonov) [inline tcp-mocker-app-test/tcp-mocker-app/docker-compose.yml]: version 2 does not support docker-network labels https://github.com/docker/compose/issues/6636

File: tcp-mocker-app-test/src/test/resources/testcontainers.properties [hunk 3]
```diff
@@ -1 +1,2 @@
-compose.container.image=docker/compose:1.23.2
\ No newline at end of file
+# org.testcontainers.utility.TestcontainersConfiguration
+socat.container.image=alpine/socat:1.7.3.4-r0
```
Comment (andreysaksonov) [inline tcp-mocker-app-test/src/test/resources/testcontainers.properties]: by default version is unpinned e.g. latest which might affect the stability of the build

File: tcp-mocker-app-test/src/test/java/io/payworks/labs/tcpmocker/TcpMockerAppDockerIT.java [hunk 4]
```diff
@@ -13,32 +16,38 @@
...
 
+    private static final Logger logger = LoggerFactory.getLogger(TcpMockerAppDockerIT.class);
+
     private static final String TCP_MOCKER_APP_SERVICE_NAME = "tcp-mocker-app";
 
     private static final int TCP_SERVICE_PORT = 10001;
     private static final int WEB_SERVICE_PORT = 8080;
 
-    private static final String TCP_MOCKER_APP_CONTAINER_TAG = Optional.ofNullable(System.getenv("PROJECT_VERSION"))
+    private static final String dockerTcpMockerAppContainerTag = Optional.ofNullable(System.getenv("PROJECT_VERSION"))
```
Comment (enrique-alonso-martin-pw) [inline tcp-mocker-app-test/src/test/java/io/payworks/labs/tcpmocker/TcpMockerAppDockerIT.java]: Why the renaming?

Comment (andreysaksonov) [reply tcp-mocker-app-test/src/test/java/io/payworks/labs/tcpmocker/TcpMockerAppDockerIT.java]: it is not a constant effectively

**Pull request merged**